build/dependencies.props

```diff
@@ -147,10 +146,8 @@
     <SystemInteractiveAsyncPackageVersion>3.1.1</SystemInteractiveAsyncPackageVersion>
     <SystemIOFileSystemAccessControlPackageVersion>4.5.0-preview2-26130-01</SystemIOFileSystemAccessControlPackageVersion>
     <SystemIOPackagingPackageVersion>4.5.0-preview2-26130-01</SystemIOPackagingPackageVersion>
-    <SystemIOPipelinesExtensionsPackageVersion>0.1.0-preview2-180130-1</SystemIOPipelinesExtensionsPackageVersion>
```

Comment:
This was removed in release/2.1, should it be removed in dev?

Reply:
Yes